addon/components/labs-ui/site-header.hbs

```diff
@@ -0,0 +1,28 @@
+{{!-- template-lint-disable no-invalid-interactive  --}}
+
+<header class="site-header" role="banner">
+  {{#if @betaNotice}}
+    <a href="https://planninglabs.nyc/" class="labs-beta-notice hide-for-print">A beta project of NYC Planning Labs</a>
+  {{else}}
+    {{yield (hash pre = (component "labs-ui/site-header-pre"))}}
+  {{/if}}
+
+  <div class="grid-x grid-padding-x">
+
+    <div {{action (mut this.closed) true preventDefault=false}} class="branding cell {{if @responsiveNav (concat @responsiveSize "-auto shrink") "auto"}}">
+      <a class="dcp-link" href="http://www1.nyc.gov/site/planning/index.page"><img class="dcp-logo" src="https://raw.githubusercontent.com/NYCPlanning/logo/master/dcp_logo_772.png" alt="NYC Planning"></a>
+      {{yield (hash title=(component "labs-ui/site-header-title"))}}
+    </div>
+
+    {{#if @responsiveNav}}
+      <div class="cell auto hide-for-{{@responsiveSize}} text-right">
+        <button {{action (mut this.closed) (not this.closed)}} class="responsive-nav-toggler hide-for-print">Menu</button>
+      </div>
+    {{/if}}
+
+    <nav {{action (mut this.closed) (not this.closed) preventDefault=false}} role="navigation" class="cell {{if @responsiveNav (concat @responsiveSize "-shrink responsive") "shrink"}} {{if (and @responsiveNav this.closed) (concat "show-for-" @responsiveSize)}} hide-for-print">
```

Comment:
AFAIU, a `@tracked` property is tracked to compute another property. You can mutate an untracked property, so this works as-is. 

But I too would like to know more about the exact requirements for using `@tracked`. [The docs](https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/ ) are confusion because they say, "Tracked properties replace computed properties." But then go on to give examples of properties that are computed by using tracked properties. 

It seems that tracked properties are like what computed properties used to take as parameters for computation. But now you don't have to pass them to the getter. 🤷‍♂